### PR TITLE
Fusion 08 update

### PR DESCRIPTION
**Pull request by omonoukpoma**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,3 +20,53 @@ As a user, I would like to fetch content from an API into my feature pack.
...
+    }).isRequired,
+  }),
+};
+```
+
+3. Within the Card component, retrieve the `storyConfig` custom field
+
+4. This `storyConfig` custom field contains `contentService` and `contentConfigValues`. Extract them from `storyConfig` 
+
+5. Now, we need to fetch the content using Fusion's `useContent`. Import `useContent` from Fusion:
```
Comment (nicrocs) [inline README.md]: Adding a link to the ALC docs could be helpful to provide context. https://redirector.arcpublishing.com/alc/arc-products/pagebuilder/fusion/documentation/recipes/fetching-content-with-hooks.md

File: README.md [hunk 2]
```diff
@@ -20,3 +20,53 @@ As a user, I would like to fetch content from an API into my feature pack.
...
+
+## New Task: User Story 09
+As a user, I would like to be able to pipe content into my feature so that it renders information from an API.
+
+## Guide
+We are going to create a new `Card` component, which will read the headline and subheadline from our content source
+
+1. Create a new *global* feature called `Card` (i.e. the `deafult.jsx` should be placed in `components/features/global/card` folder)
+
+2. Add a PageBuilder custom field that enables the user to select our article content source:
```
Comment (nicrocs) [inline README.md:36]: Adding a link to the ALC docs here should help provide context to the `contentConfig` proptype https://redirector.arcpublishing.com/alc/arc-products/pagebuilder/fusion/documentation/recipes/dynamically-configuring-content-with-hooks.md

**Review by nicrocs — CHANGES_REQUESTED** [review]
Just a couple of quick additions to link to the docs for some context. Otherwise look good!

**Comment by omonoukpoma** [comment]
@nicrocs links have been added in

**Review by nicrocs — APPROVED** [review]

**Pull request merged**